docs/src/pages/style/typography/typography.md

```diff
@@ -11,15 +11,16 @@ A [typographic scale](https://material.io/design/typography/#type-scale) has a l
 
 ## General
 
-The *Roboto* font will **not** be automatically loaded by Material-UI.
+The _Roboto_ font will **not** be automatically loaded by Material-UI.
```

Comment:
For consistency with the codebase, eventually, prettier will format it for us.

```suggestion
The *Roboto* font will **not** be automatically loaded by Material-UI.
```

Reply:
👍 I'm not entire sure where this was introduced. I just tried see if VSCode or Prettier formatted it this way, but couldn't make either swap * for _ for italics. Will revert.

Reply:
Scratch that. All of these were introduced when I run `yarn docs:typescript:formatted` (e.g. `docs/scripts/formattedTSDemos`